src/main/java/com/internet/shop/controllers/cart/CartController.java

```diff
@@ -0,0 +1,29 @@
+package com.internet.shop.controllers.cart;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.ShoppingCartService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CartController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private ShoppingCartService cartService =
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        ShoppingCart cart = cartService.getByUserId(USER_ID);
+        List<Product> productList = cart.getProducts();
+
+        req.setAttribute("cartList", productList);
```

Comment:
```suggestion
        req.setAttribute("products", productList);
```
cartList sounds like you have several shoppingCarts

Reply:
done